public/components/presence-indicator/presence-indicators.js

```diff
@@ -20,12 +20,22 @@ function wfPresenceIndicatorsDirective ($rootScope, wfPresenceService,
                     $scope.presences = _.map(
                         _.uniqBy(currentState, (s) => { return s.clientId.person.email; }),
                         (pr) => {
-                            var person = pr.clientId.person;
-                            return { indicatorText:
-                                (person.firstName.charAt(0) + person.lastName.charAt(0)).toUpperCase(),
+                            const person = pr.clientId.person;
+
+                            const presenceObject = {
+                                indicatorText:
+                                    (person.firstName.charAt(0) + person.lastName.charAt(0)).toUpperCase(),
                                 longText: [person.firstName, person.lastName].join(" "),
-                                email: person.email,
-                                status: "present" };
+                                email: person.email
+                            };
+
+                            if(currentState[0].location === "body") {
```

Comment:
I'm not sure if there are other values that `location` can have. When you click 'Save and close' in Composer, leaving the `body` location, the Presence client still registers you as being present in the Article but with an empty location value.